compose/Dockerfile

```diff
@@ -13,11 +13,15 @@ RUN apt-get update \
   && rm -rf /var/lib/apt/lists/*
 
 # Requirements are installed here to ensure they will be cached.
-COPY requirements.txt ./
+COPY ./requirements.txt ./
 RUN pip install -r requirements.txt
 
 COPY . .
 
+COPY ./compose/wait-for-mysql /wait-for-mysql
+RUN sed -i 's/\r$//g' /wait-for-mysql
+RUN chmod +x /wait-for-mysql
```

Comment:
Trenger kanskje ikke kopiere `wait-for-mysql` på nytt når det allerede gjøres på linje 19?
```suggestion
COPY . .

# Wait for the mysql-container to start
RUN sed -i 's/\r$//g' ./compose/wait-for-mysql
RUN chmod +x ./compose/wait-for-mysql
```

Reply:
Nei, du har helt rett, men docker-compose fant ikke filen da jeg ikke gjorde det selv om jeg endret alle steder. Skal gi det et nytt forsøk

Reply:
Merkelig men ikke krise, godkjenner jeg 🔥

Reply:
Nice 🔥 Det går helt fint å gjøre som du sier hvis jeg endrer line separator fra CRLF til LF manuelt i Pycharm, men mener de RUN-kommandoene skulle fikse det

Reply:
Tror compose bare bruker filen jeg har lokalt med CRLF i stedet for den jeg mener å bruke her, som skal bli konvertert til LF i bildet og kjøre fint. Fjerner jeg volumes fra docker-compose så fungerer det.

Jeg lar det stå så slipper vi å ha problemer med at docker ikke finner filen pga carriage return (CRLF) og at vi må konverte manuelt i en editor eller vha dos2unix 👍